decidim-templates/app/views/decidim/templates/admin/block_user_templates/_template_chooser.html.erb

```diff
@@ -0,0 +1,16 @@
+<% templates = Decidim::Templates::Template .where(
```

Comment:
There's a small typo here (a space) but couldn't we move this to a helper or the controller?